amplify/backend/api/colonycdapp/schema.graphql

```diff
@@ -3399,11 +3433,64 @@ type StreamingPayment @model {
   Is the stream waived?
   """
   isWaived: Boolean
+  """
+  The motions created that relate to this streaming payment, if there are any
+  """
+  motions: [ColonyMotion]
+    @hasMany(indexName: "byStreamingPaymentId", fields: ["id"])
+  """
+  List of Streaming Payment changelog entries
+  """
+  changelog: [StreamingPaymentChangelog!]
```

Comment:
Ah okay, I only added it directly to the streaming payment rather than the action because #2213 states it doesn't require a corresponding action to be created - but if you're happy to have a colony action with it then I can go ahead and make that change. 👍